internal/configs/virtualserver.go

```diff
@@ -401,14 +419,14 @@ func generateRulesRouteConfig(route conf_v1alpha1.Route, upstreamNamer *upstream
 	}
 }
 
-var specialMapParameters = map[string]bool{
-	"default":   true,
-	"hostnames": true,
-	"include":   true,
-	"volatile":  true,
-}
-
 func generateValueForRulesRouteMap(matchedValue string) (value string, isNegative bool) {
+	var specialMapParameters = map[string]bool{
```

Comment:
What is the reason to change this to inside the function? I am curious 🤔

Reply:
I simply don't like to have global variables unless there's a good reason for them. This seems only used within this func. I think it's also slightly easier to read.

Would you like me to revert this? - Why?

Reply:
it is a variable but it is a constant. we never change it. there is no need to initialize it every time that function is called.